delta/delta.go

```diff
@@ -104,6 +137,19 @@ func (d *Delta) Insert(text string, attrs map[string]interface{}) *Delta {
 	return d
 }
 
+// InsertEmbed takes a map of embeds and a map of attributes, adds them to the Delta. This
+// can be used to insert images or URLs to the Delta
+func (d *Delta) InsertEmbed(embeds map[string]interface{}, attrs map[string]interface{}) *Delta {
+	newOp := Op{
```

Comment:
the `Insert` method has this guard:

```
if len([]rune(text)) == 0 {
		return d
	}
```

we should have something similar here, where we check that the key has an actual value, if not, just return the original `d *Delta`